storage_service/locations/tests/test_package.py

```diff
@@ -238,3 +241,57 @@ def test_extract_file_aip_from_compressed_aip(self):
         output_path, extract_path = package.extract_file(extract_path=self.tmp_dir)
         assert output_path == os.path.join(self.tmp_dir, basedir)
         assert os.path.join(output_path, 'manifest-md5.txt')
+
+
+class TestTransferPackage(TestCase):
+    """Test integration of ``Package`` transfer package parsing and indexing.
+
+    It uses ``subprocess`` and ``rsync``.
+    """
+
+    def setUp(self):
+        self.tmp_dir = tempfile.mkdtemp()
+
+    def tearDown(self):
+        shutil.rmtree(self.tmp_dir)
+
+    def _rsync_fixture(self, fixture_dir, dst, make_bagit=False):
```

Comment:
Does `shutil.copytree` work instead of rsync? It seems better to avoid a dependency on rsync if we can (I know it's in basically all distros, but it's not core, and it requires the use of subprocess).

Reply:
Cool, it works.